From: "Uladzislau Rezki (Sony)" <urezki@gmail.com>
Subject: mm: vmalloc: simplify return boolean expression
Date: Thu, 20 Oct 2022 14:52:47 +0200

A return expression of the __purge_vmap_area_lazy() function can be
simplified.  Basically a boolean expression anyway is converted to boolean
values either to true or false.

Link: https://lkml.kernel.org/r/20221020125247.5053-1-urezki@gmail.com
Signed-off-by: Uladzislau Rezki (Sony) <urezki@gmail.com>
Suggested-by: Christoph Hellwig <hch@infradead.org>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Cc: Nicholas Piggin <npiggin@gmail.com>
Cc: Oleksiy Avramchenko <oleksiy.avramchenko@sony.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vmalloc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/vmalloc.c~mm-vmalloc-simplify-return-boolean-expression
+++ a/mm/vmalloc.c
@@ -1786,7 +1786,7 @@ static bool __purge_vmap_area_lazy(unsig
 
 out:
 	trace_purge_vmap_area_lazy(start, end, num_purged_areas);
-	return num_purged_areas > 0 ? true:false;
+	return num_purged_areas > 0;
 }
 
 /*
_
